Agents/ForecastingAgent/tests/test_triples/example_abox.ttl

```diff
@@ -89,5 +89,20 @@
     ts:hasCovariate :AirTemperature_1 ;
     ts:hasCovariate :IsHoliday_1 .
 
+:ForecastingModel_3 rdf:type ts:ForecastingModel ;
+    rdfs:label "Prophet" ;
+    ts:scaleData "False"^^xsd:boolean ;
+    ts:hasCovariate :Promotion_1 ;
+    ts:hasCovariate :SpecialEvent_1 .
+
+:ForecastingModel_4 rdf:type ts:ForecastingModel ;
+    rdfs:label "Prophet" ;
+    ts:scaleData "True"^^xsd:boolean ;
+    ts:hasCovariate :Promotion_1 ;
+    ts:hasCovariate :SpecialEvent_1 .
+
 :AirTemperature_1 rdf:type ems:AirTemperature .
 :IsHoliday_1 rdf:type dh:isPublicHoliday .
+
+:Promotion_1 rdf:type ems:Promotion .
```

Comment:
Sorry for being nitpicking here: `ems:Promotion` and `dh:SpecialEvent` do not exist as concept in any of our ontology (and might send a wrong signal to users). Please just create another `ems:AirTemperature` or `owl:Thing`